magento/pwa-studio
# [PWA-633] My Account: Address Book: Main View

## Description

As a shopper, when logged in and viewing the address book page I should see any of my stored addresses with any associated options. 

In-Scope: 

Create main address components for the Address book page for both desktop and mobile

Requirements: 

For each address card, shopper can view: 

Name
Address
Default Status
Edit (CTA)
Delete (CTA)
## Related Issue





* [[PWA-633](https://jira.corp.magento.com/browse/PWA-633)] My Account: Address Book: Main View

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Compare Address book UI with mocks when -
    1. User does not have any address.
    2. User has one address
    3. User has 2 Address
    4. User has Multiple address
2. Verify Default address should have  Default label and should not have delete icon.
3. Verify address with minimum and maximum characters.
4. Verify page works from both left drawer and Top right Account menu.
5. Verify Existing user address gets displayed.
6. Verify after logout from Address book page user is taken to Home page.
7. Verify Mobile scroll functionality when there are many address.
8. Verify in all supported browsers and devices.
9. Checkout Edit shipping address should show similar data as of My address page.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
